klt-cert-manager/Makefile

```diff
@@ -56,7 +59,7 @@ help: ## Display this help.
 
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) rbac:roleName=klc-cert-manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) rbac:roleName=cert-manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```

Comment:
shouldn't it be klt-* ?

Reply:
no :(  this is auto added by kubebuilder otherwise we get klt-klt